content/md/products-super-power/restore-a-product.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: restore-a-product
 themes: products-super-power
-title: Restore an **old product version**
+title: Restore a **previous product version**
```

Comment:
This article is not only about restoring the previous version but an oldest version